server/src/server.js

```diff
@@ -1065,15 +1073,16 @@ const END_POINT_SEPARATOR = "|";
 
 // Get OAuth client params for the client-side authorization flow.
 app.get("/api/fxa-oauth/login/*", async function(req, res, next) {
-  if (!req.deviceId) {
-    next(errors.missingSession());
-    return;
-  }
-
   try {
     const stateBytes = await randomBytes(32);
     let state = stateBytes.toString("hex");
-    const inserted =  await setState(req.deviceId, state);
+    const cookies = new Cookies(req, res, {keys: dbschema.getKeygrip("auth")});
```

Comment:
We should create a new scope here instead of using `"auth"`. That just requires a different string for `getKeygrip()` and then listing that string in dbschema.js

Reply:
Updated to use 'fxa-oauth' as new scope